### PR TITLE
#169257339 setup sass pre-processing

### PR DESCRIPTION
**Pull request by nshaphan**
#### What does this PR do?
setup sass pre-processing on the application

#### Description of Task to be completed?
- [x] Update webpack configurations for file loaders
- [x] Created a base folder structure for development using sass

#### How should this be manually tested?
- run npm install to install dependencies.
- run `npm run watch` to test sass setup.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#169257339](https://www.pivotaltracker.com/story/show/169257339) 

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

File: src/assets/scss/pages/_index.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@use './utilities/variables';
```
Comment (gildniy) [inline src/assets/scss/pages/_index.scss]: I think here you need to point from the upper-level folder  like this`@use '../utilities/variables';`

Comment (nshaphan) [reply src/assets/scss/pages/_index.scss]: Great!, let me work on it!

Comment (nshaphan) [reply src/assets/scss/pages/_index.scss]: Done!

File: .gitignore [hunk 2]
```diff
@@ -268,3 +268,4 @@ sketch
 
 /node_modules/
 /.idea/
+storybook-static/
```
Comment (Billmike) [inline .gitignore]: Add a new line here

File: src/assets/scss/app.scss [hunk 3]
```diff
@@ -0,0 +1 @@
+@use './pages/index';
```
Comment (Billmike) [inline src/assets/scss/app.scss]: New line here

File: src/assets/scss/base/_typography.scss [hunk 4]
```diff
@@ -0,0 +1 @@
+$font-stack: Arial, Helvetica, sans-serif;
```
Comment (Billmike) [inline src/assets/scss/base/_typography.scss]: New line here

File: src/assets/scss/pages/_index.scss [hunk 5]
```diff
@@ -0,0 +1,7 @@
+@use '../utilities/variables';
+@use '../base/typography';
+
+body {
+  color: variables.$text-color;
+  font-family: typography.$font-stack;
+}
```
Comment (Billmike) [inline src/assets/scss/pages/_index.scss]: New line here

File: src/assets/scss/utilities/_variables.scss [hunk 6]
```diff
@@ -0,0 +1 @@
+$text-color: #0747A6;
```
Comment (Billmike) [inline src/assets/scss/utilities/_variables.scss]: New line here

**Comment by nshaphan** [comment]
@Billmike New lines added

**Pull request merged**